### ISSUE TITLE
max-len indented comments

### ISSUE THREAD

**Issue by tbremer** [eslint/eslint#6322]
**What version of ESLint are you using?** `2.11.1` (from my package.json: ^2.11.1)

**What parser (default, Babel-ESLint, etc.) are you using?** `default`

**Please show your full configuration:** [full config](https://github.com/tbremer/eslint-config-tbremer/blob/master/index.js) piece concerned with:
```json
"max-len": [
  2,
  {
    "code": Infinity,
    "comments": 80,
    "tabWidth": 2
  }
],
```

**What did you do? Please include the actual source code causing the issue.** I found the issues when a single line comment was indented but longer than my set `comments` length

```javascript
function foo(something) {
  if (!something) throw new Error('something is reqquired');
  
  // this is a really long commnet this is a really long commnet this is a really long commnet this is a really long commnet
  return somethingDifferent;
}
```

_actual_ code with issue:

```javascript
export default class Maitre {
  set port(p) {
    if (this.__port__ !== undefined) throw new Error('Port should not be reassigned.');

    //this is a long comment this is a long comment this is a long comment this is a long comment this is a long commentthis is a long comment this is a long comment this is a long comment this is a long comment this is a long commentthis is a long comment this is a long comment this is a long comment this is a long comment this is a long comment
    this.__port__ = p;
  }
}
```

**What did you expect to happen?** max-len to throw an error

**What actually happened? Please include the actual, raw output from ESLint.** went a long like everything was cool.

**Comment by tbremer** [comment]
This is actually pretty easy to recreate:
```
function foo() {
  //this is a long comment this is a long comment this is a long comment this is a long comment this is a long commentthis is a long comment this is a long comment this is a long comment this is a long comment this is a long commentthis is a long comment this is a long comment this is a long comment this is a long comment this is a long comment
}
```

I could be making a bad assumption here, but this could be as simple as a check in `isFullLineComment` to trim whitespace?

**Comment by kaicataldo** [comment]
Was able to recreate locally with the latest version of ESLint.

With the provided configuration, 
```js
function foo() {
//this is a long comment this is a long comment this is a long comment this is a long comment this is a long commentthis is a long comment this is a long comment this is a long comment this is a long comment this is a long commentthis is a long comment this is a long comment this is a long comment this is a long comment this is a long comment
}
```
errors, but
```js
function foo() {
  //this is a long comment this is a long comment this is a long comment this is a long comment this is a long commentthis is a long comment this is a long comment this is a long comment this is a long comment this is a long commentthis is a long comment this is a long comment this is a long comment this is a long comment this is a long comment
}
```
does not.

**Comment by kaicataldo** [comment]
In looking over it quickly, it looks to me like a bug in `isFullLineComment` - probably the `start.column === 0` check

**Comment by tbremer** [comment]
I wasn't sure about the `=== 0` because of multiline comments, but playing with the parser a bit I saw that multiline's were broken down as well.

Thanks @kaicataldo!

**Comment by tbremer** [comment]
@kaicataldo I played around for a few minutes with removing the `start.column === 0` and it fixed the issue for me, also tried throwing comments at the end of a statement / method and that worked, and block comments held up.

I think you're assumption is correct.
